docs/source/extensions/overview_extensions.rst

```diff
@@ -34,9 +36,21 @@ of the core project:
 Community-maintained extensions
 -------------------------------
 
-The following extensions are developed and maintained by the ASReview community:
+ASReview has support for community-maintained extensions, that enable you to seemlessly integrate 
+your own code with the ASReview framework. These extensions can extend the software with new 
+classifiers, query strategies, balance strategies, and feature extraction techniques.
+
+A template for creating your own ASReview extension can be found here: `Template <https://github.com/asreview/template-extension-new-model>`__.
```

Comment:
I think I prefer to point to the documentation instead of the template. The template is limited compared what ASReview offers for developers. 

https://asreview.readthedocs.io/en/latest/API/extensions_dev.html